src/models/UserMapper.php

```diff
@@ -38,6 +38,7 @@ public function getVerboseFields()
             "username"         => "username",
             "full_name"        => "full_name",
             "twitter_username" => "twitter_username",
+            "trusted"          => "trusted"
```

Comment:
Does this need to be exposed? We don't expose whether someone is an admin, so why exposing this? Or am I missing something?